lib/pronto/github.rb

```diff
@@ -45,17 +45,13 @@ def create_pull_comment(comment)
       end
     end
 
-    def create_pull_request_review(comments)
-      return if comments.empty?
-
-      options = {
-        event: 'COMMENT',
-        accept: 'application/vnd.github.black-cat-preview+json', # https://developer.github.com/v3/pulls/reviews/#create-a-pull-request-review
-        comments: comments.map do |c|
-          { path: c.path, position: c.position, body: c.body }
-        end
-      }
-      client.create_pull_request_review(slug, pull_id, options)
+    def publish_pull_request_comments(comments)
+      comments_left       = comments.clone
```

Comment:
@dimitrovv we prefer a style of `comments_left_ = comments.clone` instead of aligning it this way.

Reply:
so I guess it would be better to leave it this way _(without alignment)_ or:

```ruby
comments_left = comments.clone
warnings_per_review = @config.warnings_per_review || comments.size
```

Reply:
Should I fix the alignment everywhere?